apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/FirestoreMultipleDbTest.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022, Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.ClientTesting;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Firestore.IntegrationTests;
+
+public class FirestoreMultipleDbTest
+{
+    private readonly MulitpleFirestoreDbFixture _fixture;
+    public FirestoreMultipleDbTest()
+    {
+        _fixture = new MulitpleFirestoreDbFixture();
+    }
+    
+    [Fact]
+    public async Task MultipleDbReadWriteTest()
+    {
+        string dbIdPrefix = IdGenerator.FromGuid(prefix: "db", separator: "-");
+        string correctDbId = $"{dbIdPrefix}-correct";
+        string incorrectDbId = $"{dbIdPrefix}-incorrect";
+        string databaseLocationId = "us-east1";
+        string databaseType = "FIRESTORE_NATIVE";
+        // Create two databases one for reading and writing, one for reading to validate that correct database should be referenced.

```

Comment:
I'd suggest adding a blank line above each of the comments, just to split the test up a bit.